src/test/java/systemtests/DeleteCommandSystemTest.java

```diff
@@ -101,11 +102,9 @@ public void delete() {
         command = DeleteCommand.COMMAND_WORD + " " + outOfBoundsIndex.getOneBased();
         assertCommandFailure(command, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
 
-        /* Case: invalid arguments (alphabets) -> rejected */
```

Comment:
is there a reason for removing this particular test case?

Reply:
Yes, because deleting using alphabets will trigger the delete by name command instead of deleting with index, so it won't be rejected.